relay-general/src/store/transactions.rs

```diff
@@ -695,7 +703,7 @@ mod tests {
                 contexts
             })),
             spans: Annotated::new(vec![Annotated::new(Span {
-                timestamp: Annotated::new(Utc.ymd(2000, 1, 1).and_hms(0, 0, 0)),
+                timestamp: Annotated::new(Utc.ymd(2000, 1, 1).and_hms(0, 0, 10)),
```

Comment:
Same consideration as above, making the test data more realistic: non-zero duration.